.cz-config.js

```diff
@@ -0,0 +1,49 @@
+const { execSync } = require('child_process');
+
+const getPackages = () => {
+  let outputPackages;
+
+  try {
+    outputPackages = execSync('npx turbo run build --dry=json');
+  } catch (error) {
+    console.info('No changed packages found.');
+    process.exit(0);
+  }
+
+  const packagesArray = JSON.parse(outputPackages.toString());
+
+  return packagesArray.packages.map((packageName) => packageName.replace('@justeattakeaway/', ''));;
+};
+
+module.exports = {
+  // We have to provide '\u0020' to some emojis due to a spacing bug with commitizen - https://github.com/commitizen/cz-cli/issues/815
+  types: [
+    { value: 'feat', name: '✨ feat: A new feature or functionality' },
+    { value: 'fix', name: '🐛 fix: A bug fix' },
+    { value: 'chore', name: '🧹 chore: Changes that affect the build system or external dependencies (e.g: typescript, webpack, eslint)' },
+    { value: 'ci', name: '⚙️ \u0020ci: Changes to our CI configuration files and scripts' },
+    { value: 'docs', name: '📚 docs: Documentation only changes' },
+    { value: 'format', name: '💅 format: Changes that do not affect the meaning of the code (e.g: white-space, linting fixes etc)' },
+    { value: 'performance', name: '🚀 performance: A code change that improves performance' },
+    { value: 'refactor', name: '♻️ \u0020refactor: A code change that neither fixes a bug nor adds a feature' },
+    { value: 'release', name: '📦 release: Release of version' },
```

Comment:
Not sure if this should be under the same category, but would this make sense potentially? (I'm mainly thinking about what we'd put things like renovate PRs under in the future)

```suggestion
    { value: 'release', name: '📦 release: Release a new version or package.json updates' },
```

Reply:
Either this, or we would just include package updates as a :chore: update potentially, any thoughts?

Reply:
My only worry is that by adding `or package.json updates`, people may end up using this category for dependency changes. Do you think it's worth adding a specific category for deps?

For stuff like renovate, i'd personally put this under `feat` / `chore` with a scope of `pie-monorepo`. Thoughts?

Reply:
Yeh, I think chore makes sense potentially (might be worth adding something to the name field there so that it's called out for package updates)